src/patternfly/demos/Toolbar/examples/Toolbar.md

```diff
@@ -435,3 +435,89 @@ import './Toolbar.css'
   {{/page-main-section}}
 {{/page-demo-default}}
 ```
+
+### Toolbar expand all (collapsed)
+```hbs isFullscreen
+{{#> page-demo-default page-demo-default--id="toolbar-expand-all-collapsed"}}
+  {{#> page-main-section}}
+    {{#> toolbar toolbar--id=(concat page-demo-default--id ' mobile-example') toolbar--modifier="pf-m-nowrap pf-m-page-insets"}}
+      {{#> toolbar-content}}
+        {{#> toolbar-content-section toolbar-content-section--modifier="pf-m-nowrap"}}
+          {{#> toolbar-group toolbar-group--modifier="pf-m-toggle-group pf-m-show"}}
+            {{> toolbar-toggle toolbar-toggle--IsExpanded="false"}}
+            {{#> toolbar-group toolbar-group--modifier="pf-m-filter-group"}}
```

Comment:
I think we only need to use an `icon-group` if `expand-all` is next to a `bulk-select`.

![Screen Shot 2020-10-20 at 11 12 49 AM](https://user-images.githubusercontent.com/5385435/96606474-517dfa80-12c5-11eb-9353-a0dc9d47fee7.png)

Otherwise, the `expand-all` can simply be a `toolbar__item`. 

**Omitting the `icon-group` wrapper would result in this:**

![Screen Shot 2020-10-20 at 11 15 22 AM](https://user-images.githubusercontent.com/5385435/96606769-a1f55800-12c5-11eb-8287-64e49d511b65.png)

@mcarrano WDYT?

Reply:
@mattnolting updated the examples to reflect this spacing, looks a lot better.